app/jobs/event/apply_fee_waiver_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Event
+  class ApplyFeeWaiverJob < ApplicationJob
+    queue_as :low
+
+    DATE_LOCK = Date.new(2025, 11, 1)
+
+    def perform
+      Event.find_each do |event|
+        process_event(event)
+      end
+    end
+
+    private
+
+    def process_event(event)
+      active_teen_count = event.users.active_teenager.count
+
+      if active_teen_count >= 5 && event.fee_waiver_eligible && Date.current < DATE_LOCK
+        plan_type =
+          if active_teen_count >= 10
+            Event::Plan::Standard::FeeWaived
+          else
+            Event::Plan::Standard::ThreePointFive
```

Comment:
```suggestion
            Event::Plan::FeeWaived
          else
            Event::Plan::ThreePointFive
```
The correct names for these classes don't have Standard in them.

<img width="288" height="177" alt="Image" src="https://github.com/user-attachments/assets/42d7bcef-fcd2-480b-a52c-66f4f251cb77" />